aten/src/ATen/native/cuda/Dropout.cu

```diff
@@ -117,12 +227,29 @@ fused_dropout_cuda(const Tensor& self, double p, Generator * gen_){
       self_info.collapseDims();
       ret_info.collapseDims();
       mask_info.collapseDims(); //ret and mask are collapsed to 1d contiguous tensor
-      switch (self_info.dims) {
-        case 1:
-            fused_dropout_kernel<scalar_t, accscalar_t, unsigned int, 1><<<grid, dim_block, 0, at::cuda::getCurrentCUDAStream()>>>(self_info, ret_info, mask_info, nelem, pa, rng_engine_inputs);
-            break;
-        default:
-            fused_dropout_kernel<scalar_t, accscalar_t, unsigned int, -1><<<grid, dim_block, 0, at::cuda::getCurrentCUDAStream()>>>(self_info, ret_info, mask_info, nelem, pa, rng_engine_inputs);
+
+      int vec_size = get_vector_size<scalar_t>(self, ret, mask);
+
+      if (vec_size > 1) {
+        switch (vec_size) {
+         case 8:
```

Comment:
What's that? You don't generate enough randoms to handle that (or I'm blind)

Reply:
From side-conversation, this was vestigial code that I forgot to remove - `vec_size=8` is not supported for now.